Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
+using System;
+using Newtonsoft.Json;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.EntityFrameworkCore; 
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Configuration.AddJsonFile("appsettings.json");

```

Comment:
I believe, asp.net automatically includes the appsettings.json file.

Also, you will need to fetch prod config from aws system manager.

Reply:
I think I didn't get this "you will need to fetch prod config from aws system manager." so explain it pls.

Reply:
Created a separate task for it. Will do it in a separate change.